server/routes/createGoal/createGoalFormValidator.test.ts

```diff
@@ -1,10 +1,16 @@
 import type { CreateGoalForm } from 'forms'
 import validateGoalTitle from '../../validators/goalTitleValidator'
+import goalTargetCompletionDateValidator from '../../validators/goalTargetCompletionDateValidator'
 import validateCreateGoalForm from './createGoalFormValidator'
 
 jest.mock('../../validators/goalTitleValidator')
+jest.mock('../../validators/goalTargetCompletionDateValidator')
+
 describe('createGoalFormValidator', () => {
   const mockedValidateGoalTitle = validateGoalTitle as jest.MockedFunction<typeof validateGoalTitle>
+  const mockedValidateTargetCompletionDate = goalTargetCompletionDateValidator as jest.MockedFunction<
```

Comment:
The `createGoalFormValidator` unit test was failing because we were not mocking it's `goalTargetCompletionDateValidator` dependency. Therefore it was using the real `goalTargetCompletionDateValidator` validator, and with a date in some of the tests of 26 Feb 2024 (ie. today) it started failing.
Changed the test to mock `goalTargetCompletionDateValidator`

Dont worry! We are not testing mocks here! 🤣 
The unit under test `createGoalFormValidator` calls 2 other validators internally (`mockedValidateGoalTitle` and `mockedValidateTargetCompletionDate`) and returns values based on the return values from the 2 dependencies.

The `mockedValidateGoalTitle` was already mocked; we just need to mock `mockedValidateTargetCompletionDate`

The 2 dependant validators (`mockedValidateGoalTitle` and `mockedValidateTargetCompletionDate`) are unit tests properly themselves, so we have correct and adequate cover 👍